src/move.c

```diff
@@ -238,7 +238,9 @@ char MoveClient(ClientNode *np, int startx, int starty)
             }
          }
 
-         if(!doMove && (abs(np->x - oldx) > MOVE_DELTA
+         if(flags != MAX_NONE)
+            doMove = 0;
```

Comment:
When move status windows are enabled, this causes a bug where the window will stick around.  You probably need to do something like `DestroyMoveWindow()`.  I'm not sure setting `doMove` is the right approach though as it indicates whether the window has indeed moved.  Here the window certainly has moved, so setting `doMove` to 0 may cause other problems as well (such as not updating certain window properties).